src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.loop {
+  position: absolute;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.bigForm {
+  position: relative;
+  margin: 20px 0;
+}
+
+.searchBig {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  font-family: Avenir, serif;
+  font-weight: 400;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 10%);
+  border-radius: 4px;
+}
+
+input::placeholder {
+  font-family: Avenir, serif;
+  color: #3d4e61;
+}
+
+.searchBig:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 20%);
+}
+
+.searchBig:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-family: AvenirHeavy, serif;
+  border-radius: 3px;
+}
+
+.loopSmall {
+  position: absolute;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.smallForm {
+  position: relative;
+}
+
+.searchSmall {
+  box-sizing: border-box;
+  height: 42px;
+  width: 100%;
+  font-weight: 300;
+  padding-left: 33px;
+  line-height: 19px;
+  font-family: inherit;
+  font-size: 14px;
+  color: #3d4e61;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 10%);
+  border-radius: 4px;
+}
```

Comment:
Check the 2nd point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)